src/main/resources/com/google/api/codegen/py/docs/conf.py.snip

```diff
@@ -317,5 +317,4 @@
   napoleon_use_ivar = False
   napoleon_use_param = True
   napoleon_use_rtype = True
-
```

Comment:
I tested this with artman, and @geigerj is correct and I am incorrect. The `.baseline` files do have two newlines, but one of them does not appear in the output files.